tests/integration/cloud/rest_contracts/conftest.py

```diff
@@ -0,0 +1,148 @@
+from __future__ import annotations
+
+import enum
+import os
+import pathlib
+import subprocess
+from typing import TYPE_CHECKING, Callable, Final, Union
+
+import pytest
+from pact import Consumer, Pact, Provider
+from pact.matchers import Matcher  # noqa: TCH002
+from typing_extensions import Annotated  # noqa: TCH002
+
+from great_expectations.compatibility import pydantic
+from great_expectations.core.http import create_session
+
+if TYPE_CHECKING:
+    from requests import Session
+
+PACT_MOCK_HOST: Final[str] = "localhost"
+PACT_MOCK_PORT: Final[int] = 9292
+PACT_DIR: Final[str] = str(
+    pathlib.Path(pathlib.Path(__file__).parent, "pacts").resolve()
+)
+
+
+class RequestMethods(str, enum.Enum):
+    DELETE = "DELETE"
+    GET = "GET"
+    PATCH = "PATCH"
+    POST = "POST"
+    PUT = "PUT"
+
+
+@pytest.fixture
+def session() -> Session:
+    return create_session(access_token=os.environ.get("GX_CLOUD_ACCESS_TOKEN"))
```

Comment:
Could we add some error handling to panic if we don't have this env var set? Maybe an informative message for devs down the line?

Reply:
✅